test/fork/SNstCalls.t.sol

```diff
@@ -5,29 +5,29 @@ import "test/fork/ForkTestBase.t.sol";
 
 contract EthereumControllerSwapNSTToSNSTFailureTests is ForkTestBase {
```

Comment:
Name should be changed to `EthereumControllerDepositToSNSTFailureTests`.

Reply:
Fixed